front_end/src/app/(main)/questions/[id]/components/forecast_maker/forecast_maker_conditional/forecast_maker_conditional_binary.tsx

```diff
@@ -24,6 +26,7 @@ import ConditionalForecastTable, {
 } from "../conditional_forecast_table";
 import PredictButton from "../predict_button";
 import ScoreDisplay from "../resolution/score_display";
+import { useServerAction } from "@/hooks/use_server_action";
```

Comment:
There is eslint error for this import:

<img width="797" alt="image" src="https://github.com/user-attachments/assets/fc8b46da-1ec5-4fe4-aaa9-b1bec07e2f43">

Could you, please, adjust your editor to automatically run eslint and fix error when needed? Additionally, could you please run linter and make sure all errors are fixed before merging?